packages/esm-patient-medications-app/src/api/order-config.ts

```diff
@@ -9,9 +9,16 @@ import {
   type QuantityUnit,
 } from '../types';
 
+export interface Names {
```

Comment:
Interface names should generally be singular.

```suggestion
export interface ConceptName {
```